seeds/daos/daos_addresses_seed.csv

```diff
@@ -0,0 +1,9 @@
+blockchain,dao_creator_tool,dao,dao_wallet_address,created_date 
```

Comment:
I think the [test failure](https://github.com/duneanalytics/spellbook/actions/runs/3354183001/jobs/5557561271) may be caused by the blank ` `  in the ending of this line.
```suggestion
blockchain,dao_creator_tool,dao,dao_wallet_address,created_date
```

Reply:
omg, how did you find that? you're legend ser!

Reply:
@Hosuke holy cow yes what a legend